mitodl/mit-open
# update logos

### What are the relevant tickets?
closes https://github.com/mitodl/hq/issues/4898

### Description (What does it do?)
Updates the logos in the resource view with better sized logos

### Screenshots (if appropriate):
<img width="1695" alt="Screenshot 2024-07-22 at 7 24 18 PM" src="https://github.com/user-attachments/assets/0f24b0f1-0dc1-4b3c-881a-1d62ad49ad2a">
<img width="329" alt="Screenshot 2024-07-22 at 7 24 33 PM" src="https://github.com/user-attachments/assets/eb0c8ea6-1cb0-42ec-82f8-63e2f5c300f7">
<img width="1695" alt="Screenshot 2024-07-23 at 10 58 43 AM" src="https://github.com/user-attachments/assets/9d7a2f5e-fec8-4659-887d-e89ab10528e8">
<img width="328" alt="Screenshot 2024-07-23 at 10 58 59 AM" src="https://github.com/user-attachments/assets/e0b06de2-ae02-430b-89f1-84be08712290">

<img width="1694" alt="Screenshot 2024-07-22 at 7 27 07 PM" src="https://github.com/user-attachments/assets/8298d677-ad95-4adc-b376-d84a8a69eafa">
<img width="327" alt="Screenshot 2024-07-22 at 7 27 20 PM" src="https://github.com/user-attachments/assets/a2a7432d-4adc-4b12-b330-79313b0c0984">
<img width="1700" alt="Screenshot 2024-07-22 at 7 29 12 PM" src="https://github.com/user-attachments/assets/7c090eac-5934-4326-99d3-4b08c04d001f">
<img width="327" alt="Screenshot 2024-07-22 at 7 29 24 PM" src="https://github.com/user-attachments/assets/da6ba6ac-34c7-4cb6-8207-e897ecd3c0b6">
<img width="1699" alt="Screenshot 2024-07-22 at 7 30 35 PM" src="https://github.com/user-attachments/assets/0dbdf150-9913-431a-82c0-2f5b63cdc49f">
<img width="327" alt="Screenshot 2024-07-22 at 7 30 51 PM" src="https://github.com/user-attachments/assets/8bcf2037-213c-4bea-a63b-60464cf4c321">
<img width="1695" alt="Screenshot 2024-07-22 at 7 31 56 PM" src="https://github.com/user-attachments/assets/866be7ad-1f27-4fb5-9938-0ba379f3fa0b">
<img width="325" alt="Screenshot 2024-07-22 at 7 32 06 PM" src="https://github.com/user-attachments/assets/70d3e1df-194e-4f66-9dba-20e51eece459">

### How can this be tested?
Go to the search page and verify that the platform logos for the platforms in the issue look good. You can test this easier by adding platform=platform_code found at https://api.mitopen-rc.odl.mit.edu/api/v1/schema/redoc/#tag/learning_resources/operation/learning_resources_list to the url

For example http://localhost:8062/search?platform=see
